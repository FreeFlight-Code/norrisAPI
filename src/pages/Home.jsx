import React from 'react';
import Form from '../components/Form';

class Home extends React.Component {

	render() { 

		console.log(this.state)
		return (

      <div className="page home">
        Home Page
		<Form/>
      </div>
    );
	}
}
 
export default Home;