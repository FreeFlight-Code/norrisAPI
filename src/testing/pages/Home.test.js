import React from 'react';
import Home from '../../pages/Home';
import renderer from 'react-test-renderer';

jest.mock('../../components/LoginForm.jsx')

describe (' The Home Page... ', () => {
	it("renders correctly", () => {
	const tree = renderer.create(< Home />).toJSON();
	expect(tree).toMatchSnapshot();
	});
	//Does homepage render Login component
})

