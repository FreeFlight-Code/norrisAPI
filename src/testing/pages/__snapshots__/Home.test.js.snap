// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` The Home Page...  renders correctly 1`] = `
<div
  className="page home"
>
  <h3>
    Visit our pages above:
  </h3>
  <h4>
    Home Page allows you to log in.
  </h4>
  <h4>
    Jokes Page allows you to see the jokes you have viewed.
  </h4>
  <h4>
    Categories Page allows you to view the categories of jokes
  </h4>
  <h4>
    Search Page allows you to search jokes by key word.
  </h4>
</div>
`;
